### ISSUE TITLE
MavlinkPassthrough support in language wrappers

### ISSUE THREAD

**Issue by david6096** [mavlink/MAVSDK#2161]
I want to process custom mavlink message from android platform,but MavlinkPassthrough plugin not exported to mavsdkserver through protobuf rgpc.any scheduling for this?

**Comment by JonasVautherin** [comment]
It is not scheduled. @divyanshupundir is using https://github.com/divyanshupundir/mavlink-kotlin in parallel to MAVSDK-Java, I think.

If you only want to do raw mavlink you can look directly into mavlink-kotlin, I would say.

**Comment by julianoes** [comment]
This would be nice and is something I have planned to do. However, it's not exactly my day job, so if you want to accelerate it, consider [throwing something in the hat](https://github.com/sponsors/julianoes).

**Comment by julianoes** [comment]
Duplicate of #1459 .
